aksel.nav.no/website/app/dev/(designsystemet)/grunnleggende/styling/tokens-darkside/page.tsx

```diff
@@ -0,0 +1,105 @@
+import { Metadata, ResolvingMetadata } from "next/types";
+import React from "react";
+import { ClockDashedIcon } from "@navikt/aksel-icons";
+import {
+  BodyLong,
+  BodyShort,
+  Detail,
+  HStack,
+  Heading,
+  Link,
+  Tag,
+  VStack,
+} from "@navikt/ds-react";
+import { FigmaIcon, GithubIcon } from "@/assets/Icons";
+import TokenTableOfContents from "../../../../../../components/token-docs/TokenTableOfContents";
+import TokensPage from "../../../../../../components/token-docs/TokensPage";
+import { DesignsystemetPageLayout } from "../../../_ui/DesignsystemetPage";
+import { getStaticParamsSlugs } from "../../../slug";
+
+type Props = {
+  params: Promise<{ slug: string[] }>;
+};
+
+export async function generateMetadata(
+  { params }: Props,
+  parent: ResolvingMetadata,
+): Promise<Metadata> {
+  await params;
+
+  const ogImages = (await parent).openGraph?.images || [];
+
+  return {
+    title: "Design tokens",
+    description:
+      "Ved å bruke design tokens sørger vi for at både designere og utviklere arbeider etter de samme reglene og retningslinjene. " +
+      "Dette forenkler vedlikeholdet av designet og sikrer en helhetlig visuell fremstilling på tvers av produkter.",
+    openGraph: {
+      images: ogImages,
+    },
+  };
+}
+
+export async function generateStaticParams() {
+  return await getStaticParamsSlugs("ds_artikkel");
+}
+
+const Page = async () => {
+  return (
+    <DesignsystemetPageLayout layout="with-toc">
+      <VStack gap="space-40">
+        <VStack gap="space-24">
+          <VStack>
+            <Detail
+              data-color-role="brand-blue"
+              textColor="subtle"
+              weight="semibold"
+              uppercase
+            >
+              Styling
+            </Detail>
+            <Heading level="1" size="xlarge">
+              Tokenoversikt
+            </Heading>
+            <BodyLong size="large">
+              Tokens er alle farger, avstander, typografi, osv. som vi bruker i
+              komponenter og layout.
+            </BodyLong>
+          </VStack>
+          <VStack gap="space-16">
+            <HStack gap="space-8" align="center">
+              <Tag variant="info" size="small">
+                Stabil
+              </Tag>
+              <BodyShort size="medium">Oppdatert 22. april 2025</BodyShort>
```

Comment:
Hvordan vil vi håndtere til som "oppdatert" for denne siden? Utelate det? Hente fra GitHub-historikk? Bytte ut med "Gjelder versjon x.y.z av npm-pakkene"?

Reply:
Fjerner tag og "oppdatert", siden de ikke trengs